app/src/main/java/com/mapbox/navigation/examples/standalone/isochrone/IsochroneRangeInMeters.kt

```diff
@@ -0,0 +1,103 @@
+package com.mapbox.navigation.examples.standalone.isochrone
+
+import android.annotation.SuppressLint
+import android.graphics.Color
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.lifecycleScope
+import com.mapbox.api.isochrone.IsochroneCriteria
+import com.mapbox.api.isochrone.MapboxIsochrone
+import com.mapbox.geojson.Point
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.MapView
+import com.mapbox.maps.MapboxMap
+import com.mapbox.maps.Style
+import com.mapbox.maps.extension.style.layers.addLayer
+import com.mapbox.maps.extension.style.layers.generated.fillLayer
+import com.mapbox.maps.extension.style.sources.addSource
+import com.mapbox.maps.extension.style.sources.generated.GeoJsonSource
+import com.mapbox.maps.extension.style.sources.generated.geoJsonSource
+import com.mapbox.maps.extension.style.sources.getSourceAs
+import com.mapbox.navigation.examples.R
+import com.mapbox.navigation.examples.databinding.ActivityIsochroneRangePreviewBinding
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.launch
+
+class IsochroneRangeInMeters : AppCompatActivity() {
+
+    private val TAG = "IsochroneRangeInMeters"
+
+    /**
+     * Bindings to the example layout.
+     */
+    private lateinit var binding: ActivityIsochroneRangePreviewBinding
+
+    /**
+     * Mapbox Maps entry point obtained from the [MapView].
+     * You need to get a new reference to this object whenever the [MapView] is recreated.
+     */
+    private val mapboxMap: MapboxMap by lazy {
+        binding.mapView.getMapboxMap()
+    }
```

Comment:
```suggestion
    private var mapboxMap: MapboxMap? = null
```

onDestroy should still null out this object. "loadStyle" is technically async and _could_ return after the Activity is destroyed.